dockstore-integration-testing/src/test/java/io/dockstore/client/cli/MetadataIT.java

```diff
@@ -49,6 +49,16 @@ public void testValidClientVersion() {
         Assert.assertEquals(HttpStatus.OK_200, response.getStatusCode());
     }
 
+    @Category(MetadataResource.class)
```

Comment:
was like this already, but I don't think this category is necessary in this file 
FYI @fhembroff  in case it affects anything (I suspect not and it is just redundant)

Reply:
Removed them